airflow/providers/google/cloud/operators/gcs.py

```diff
@@ -795,11 +795,11 @@ def execute(self, context: Context) -> list[str]:
             orig_end = context["data_interval_end"]
         except KeyError:
             orig_start = pendulum.instance(context["execution_date"])
-            following_execution_date = context["dag"].following_schedule(context["execution_date"])
-            if following_execution_date is None:
-                orig_end = None
+            next_dagrun = context["dag"].next_dagrun_info(last_automated_dagrun=None, restricted=False)
```

Comment:
I guess we need to add a compatibility code here.

Reply:
No additional compat needed, just the pytest mock needs to be adjusted accordingly...

Reply:
Yep.